ffi/passmanagers.cpp

```diff
@@ -333,12 +345,14 @@ LLVMPY_AddLoopUnrollAndJamPass(LLVMPassManagerRef PM) {
     LLVMAddLoopUnrollAndJamPass(PM);
 }
 
+#if LLVM_VERSION_MAJOR < 15
 API_EXPORT(void)
 LLVMPY_AddLoopUnswitchPass(LLVMPassManagerRef PM, bool optimizeForSize,
```

Comment:
For reference the loop unswitch pass has been renamed to `SimpleLoopUnswitchPass` in the new pass manager (https://reviews.llvm.org/D124376). Naturally to enable this we need to move to NPM so it makes sense that this is disabled for now but it can come back with NPM.

Reply:
Actually `createSimpleLoopUnswitchLegacyPass` still exists in LLVM-15 so it can be dropped in here without issue.

Reply:
```
API_EXPORT(void)
LLVMPY_AddLoopUnswitchPass(LLVMPassManagerRef PM, bool optimizeForSize,
                           bool hasBranchDivergence) {
#if LLVM_VERSION_MAJOR < 15
    unwrap(PM)->add(
        createLoopUnswitchPass(optimizeForSize, hasBranchDivergence));
#else
    unwrap(PM)->add(
        createSimpleLoopUnswitchLegacyPass(optimizeForSize));
#endif
}
```

Reply:
Thanks for finding this, I have updated the code.